charts/enterprise/authelia/questions.yaml

```
@@ -405,21 +423,21 @@ questions:
                       description: "The attribute holding the username of the user"
                       schema:
                         type: string
-                        default: ""
+                        default: "uid"
                         required: true
                     - variable: additional_users_dn
                       label: "Additional Users DN"
                       description: "An additional dn to define the scope to all users."
                       schema:
                         type: string
-                        default: "OU=Users"
+                        default: "OU=people"
                         required: true
                     - variable: users_filter
                       label: "Users Filter"
                       description: "The groups filter used in search queries to find the groups of the user."
                       schema:
                         type: string
-                        default: ""
+                        default: "(&(|({username_attribute}={input})({mail_attribute}={input}))(objectClass=person))"
```

```suggestion
                        default: ""
```
User and ldap backend dependend